api-client/src/runs/types.ts

```diff
@@ -130,9 +130,16 @@ export interface LabwareOffsetCreateData {
   vector: VectorOffset
 }
 
-export interface RunTimeParameterCreateData {
-  [key: string]: string | boolean | number
+interface FileRunTimeParameterCreateData {
+  [key: string]: string | number | boolean
 }
+interface ValueRunTimeParameterCreateData {
+  [key: string]: { id: string }
+}
```

Comment:
nit (I think when Brian's pr merges in, this might turn into a lint error too?) https://github.com/Opentrons/opentrons/pull/15402

```suggestion
interface ValueRunTimeParameterCreateData {
  Record<string, { id: string }>
}
```